r-notebook/Dockerfile

```
@@ -35,16 +35,15 @@ RUN conda install --quiet --yes \
     'r-htmlwidgets=1.5*' \
     'r-irkernel=1.1*' \
     'r-nycflights13=1.0*' \
-    'r-plyr=1.8*' \
     'r-randomforest=4.6*' \
     'r-rcurl=1.98*' \
-    'r-reshape2=1.4*' \
-    'r-rmarkdown=2.1*' \
+    'r-rmarkdown=2.2    *' \
```

You should remove the trailing spaces between the version and the wildcard.
```suggestion
    'r-rmarkdown=2.2*' \
```

I think it's the cause of the error you see in the test. It works if you specify it without trailing spaces.

```bash
2020-06-06 06:19:21 [    INFO] Trying to import rmarkdown (test_packages.py:136)
2020-06-06 06:19:22 [    INFO] Trying to import base (test_packages.py:136)
PASSED  
```